metadata-ingestion/tests/integration/powerbi/golden_test_admin_access_not_allowed.json

```diff
@@ -6,7 +6,7 @@
     "aspectName": "corpUserKey",
     "aspect": {
         "json": {
-            "username": "User1@foo.com"
+            "username": "urn:li:corpuser:users.User1@foo.com"
```

Comment:
ownership should not break the old golden files. The  code should be backward compatible.